src/lib/Security/Authentication/RedirectToDashboardAuthenticationSuccessHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Security\Authentication;
+
+use eZ\Publish\Core\MVC\Symfony\Security\Authentication\DefaultAuthenticationSuccessHandler;
+use EzSystems\EzPlatformAdminUi\Specification\SiteAccess\IsAdmin;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Http\HttpUtils;
+use Symfony\Component\Security\Http\Util\TargetPathTrait;
+
+class RedirectToDashboardAuthenticationSuccessHandler extends DefaultAuthenticationSuccessHandler
+{
+    use TargetPathTrait;
+
+    /** @var array */
+    private $siteAccessGroups;
+
+    /**
+     * @param HttpUtils $httpUtils
+     * @param array $options
+     * @param array $siteAccessGroups
+     */
+    public function __construct(
+        HttpUtils $httpUtils,
+        array $options = [],
+        array $siteAccessGroups = []
+    ) {
+        parent::__construct($httpUtils, $options);
+        $this->siteAccessGroups = $siteAccessGroups;
+    }
+
+    /**
+     * Builds the target URL according to the defined options.
+     * Overwrites default page after login for admin siteaccess.
+     *
+     * @param Request $request
+     *
+     * @return string
+     *
+     * @throws \EzSystems\EzPlatformAdminUi\Exception\InvalidArgumentException
+     */
+    protected function determineTargetUrl(Request $request)
+    {
+        if ((new IsAdmin($this->siteAccessGroups))->isSatisfiedBy($request->attributes->get('siteaccess'))) {
+            $this->options['default_target_path'] = 'ezplatform.dashboard';
+            $target = $this->getTargetPath($request->getSession(), $this->providerKey);
+            if (null !== $target && 1 === count(explode('/', trim(parse_url($target)['path'], '/')))) {
```

Comment:
I would remove it. I can imagine use cases where `/` is a valid target.

(Because Admin is a SiteAccess it can be configured with `Host` matcher, and `/` would lead to a ContentTree root of `admin` SiteAccess.)

Reply:
That's true, however desired outcome is to always redirect to dashboard, even when you try to get to `/` first. (That is how it was working till now, and @lserwatka wanted to keep it this way).

Reply:
@ViniTou your feedback was it was difficult to change it. If you can redirect it to Dashboard and correct Behat test then this would be more desire behaviour imho.

Reply:
@lserwatka 
It was difficult to change outside this handler, now it is not a problem that's why I did it.